crates/reth/node/src/validator.rs

```diff
@@ -0,0 +1,57 @@
+use std::sync::Arc;
+
+use reth::builder::{components::EngineValidatorBuilder, BuilderContext};
+use reth_chainspec::ChainSpec;
+use reth_node_api::{
+    validate_version_specific_fields, EngineApiMessageVersion, EngineObjectValidationError,
+    EngineTypes, EngineValidator, FullNodeTypes, NodeTypesWithEngine, PayloadOrAttributes,
+};
+
+use crate::{StrataEngineTypes, StrataPayloadAttributes};
+
+/// Strata engine validator
+#[derive(Debug, Clone)]
+pub struct StrataEngineValidator {
+    chain_spec: Arc<ChainSpec>,
```

Comment:
We don't need to `Arc` it.
Just say 
```suggestion
    chain_spec: &ChainSpec,
```

Then the implementer deals with with `Arc`ing or not.

Example https://github.com/alpenlabs/strata/blob/f40ecce865a134511f0dc7b70c08eecce7449551/crates/consensus-logic/src/duty/worker.rs#L112

Reply:
Ok